src/App.js

```diff
@@ -1,24 +1,45 @@
-import logo from './logo.svg';
+import { NavLink, Route } from 'react-router-dom';
+import { BrowserRouter, Switch } from 'react-router-dom';
 import './App.css';
+import Fourtet from './views/fourtet';
+import Main from './views/main';
+import Movies from './views/movies';
+import Shopping from './views/shopping';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
+      <div className="App">
+        <NavLink to="/fourtet">fourtet</NavLink>
+        <br></br>
+        <NavLink to="/shopping">shopping</NavLink>
+        <br></br>
+        <NavLink to="/movies">movies</NavLink>
+        <br></br>
+        <NavLink to="/">main</NavLink>
+        <br></br>
```

Comment:
We will want to move the NavLinks into their own component (Navbar) and render the component in App.js.